src/Microsoft.Azure.WebJobs.ServiceBus/Config/IHostBuilderExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Azure.WebJobs.ServiceBus;
+using Microsoft.Azure.WebJobs.ServiceBus.Config;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using WebJobs.ServiceBus.Config;
+
+namespace Microsoft.Extensions.Hosting
+{
+    public static class IHostBuilderExtensions
+    {
+        public static IHostBuilder AddServiceBus(this IHostBuilder builder)
```

Comment:
Since we're establishing a new pattern for these extensions, can we make the ServiceBus and new Storage extensions consistent?

- put the storage builder extension in a Config directory like ServiceBus
- rename AddStorageBindings to AddStorage (consistent with AddServiceBus). In general, these new Add methods map to my old UseServiceBus etc. methods

Reply:
For the first item, I'm moving all host builder extensions all moved to the root of the project (we could have something like DependencyInjection or Hosting, not necessary for now)